theme/includes/classes/blocks.php

```diff
@@ -20,7 +20,8 @@ function __construct($editor_script_slug, $settings=array()) {
 
     $this->iconsUsed = [
       'ucd-public:fa-star',
-      'ucd-public:fa-circle-chevron-right'
+      'ucd-public:fa-circle-chevron-right',
```

Comment:
Adding it this way, will load the icon on every single page even if the block isn't used.

Instead, create a block transformation where you set the `icon` attribute to `ucd-public:fa-circle-exclamation`